lib/ansible/plugins/connection/winrm.py

```diff
@@ -47,7 +47,7 @@
     from winrm import Response
     from winrm.protocol import Protocol
 except ImportError:
-    raise AnsibleError("winrm is not installed")
+    raise AnsibleError("winrm or requests is not installed")
```

Comment:
Perhaps it would be a good idea to display the actual `message` from `ImportError` also?  Then you would actually get a better understanding of what was missing, instead of having to try and install everything.